### PR TITLE
Top peers component

### PR DESCRIPTION
**Pull request by Mark-Adaptive**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -150,6 +151,17 @@ app.get("/api/history/:symbol", async (req, res) => {
   }
 });
 
+app.get("/api/peers/:symbol", async (req, res) => {
+  console.log("app.get peers");
+  const symbol = req.params.symbol;
+  try {
+    const peersData = await fetchWrapper(iex.peers, symbol, 5);
```
Comment (jrothweiler) [inline server/index.js:158]: I think this 5 is doing nothing, can we remove it?

File: src/components/topPeers.js [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
+    <DisplayWrapper height={props.height}>
+      <TitleHeader>TOP PEERS</TitleHeader>
+      {peersInfo &&
+        peersInfo.map((peer) => (
+              <Text key={peer} className="topPeer" display="inline-block" variant="secondary" onClick={() => handleSearch(peer)}>
+                {peer}&nbsp;&nbsp;&nbsp;
+              </Text>
+        ))
+      }
+      <Text className="topPeer" display="inline-block" variant="secondary" onClick={() => handleSearch("WORK")}>
```
Comment (jrothweiler) [inline src/components/topPeers.js:26]: Maybe just put a comment here that this one is just here for demo purposes, to explain the reasoning for hardcoding one?

File: src/components/topPeers.js [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
-  const statInfo = useSelector(statsSelector);
   return (
-    <DisplayWrapper width="25%" mr="auto" ml="auto">
-      <TitleHeader>Top Peers</TitleHeader>
+    <DisplayWrapper height={props.height}>
+      <TitleHeader>TOP PEERS</TitleHeader>
+      {peersInfo &&
+        peersInfo.map((peer) => (
+              <Text key={peer} className="topPeer" display="inline-block" variant="secondary" onClick={() => handleSearch(peer)}>
+                {peer}&nbsp;&nbsp;&nbsp;
```
Comment (jrothweiler) [inline src/components/topPeers.js:22]: What is the purpose of these &nbsp; ?

File: server/index.js (hunk 1 above)
Comment (Mark-Adaptive) [reply server/index.js:158]: Yeah I'm actually not sure what that last argument does, I just took it from a different await fetchWrapper call you had. I'll remove it.

File: src/components/topPeers.js (hunk 2 above)
Comment (Mark-Adaptive) [reply src/components/topPeers.js:26]: Sure I'll add a comment

File: src/components/topPeers.js (hunk 3 above)
Comment (Mark-Adaptive) [reply src/components/topPeers.js:22]: Of what specifically?

**Pull request merged**